lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupNetworkObserver.cs

```diff
@@ -30,19 +28,21 @@ namespace Org.Apache.REEF.Network.Group.Task.Impl
 {
     /// <summary>
     /// Handles incoming messages sent to this Communication Group.
+    /// Writable version
     /// </summary>
-    public class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
+    [Obsolete("Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295 ", false)]
+    public sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
```

Comment:
You can put it as Obsolete if we already have another corresponding class. If not, you can just put TODO.